src/wavelet.cpp

```diff
@@ -120,6 +120,8 @@ Wavelet::Wavelet(const std::string   & fileName,
   }
   formats_       = modelSettings->getWaveletFormatFlag();
   waveletLength_ = findWaveletLength(modelSettings->getMinRelWaveletAmp(),0.0f);
+  //xxxxx  waveletLength_ = findWaveletLength(modelSettings->getMinRelWaveletAmp(),0.0f);
```

Comment:
On 08/24/2016 01:57 PM, hgolsen wrote:
> In src/wavelet.cpp
> <https://github.com/CRAVA/crava/pull/244#discussion_r76042784>:
>
>> @@ -120,6 +120,8 @@ Wavelet::Wavelet(const std::string   & fileName,
>>    }
>>    formats_       = modelSettings->getWaveletFormatFlag();
>>    waveletLength_ = findWaveletLength(modelSettings->getMinRelWaveletAmp(),0.0f);
>> +  //xxxxx  waveletLength_ = findWaveletLength(modelSettings->getMinRelWaveletAmp(),0.0f);
>
> Ønsker du sjekke inn denne linja?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CRAVA/crava/pull/244/files/66b75264a9b19c83d0b67b9ca952f0488c9f2f19#r76042784>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACDpfZCB5HvLptVrHR8OCxRWSA_oZVztks5qjDGcgaJpZM4Jr7PS>.
>

Ja, blir fjernet i neste runde ...